### PR TITLE
Fixed fogbench issues

### PR DESCRIPTION
**Pull request by izoratti-dianomic**
As from 170809.00

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -183,8 +162,8 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'FogLAMP', 'FogLAMP Documentation',
-     [author], 'FogLAMP', 'The FogLAMP project',
+    (master_doc, 'FogLAMP', u'FogLAMP Documentation',
+     author, 'FogLAMP', 'One line description of project.',
```
Comment (MarkRiddoch) [inline docs/conf.py]: Should add a description here

File: docs/conf.py [hunk 2]
```diff
@@ -47,18 +45,18 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'FogLAMP'
-copyright = '2017 OSIsoft, LLC'
-author = 'OSIsoft, LLC'
+project = u'FogLAMP'
+copyright = u'2017, Dianomic Systems'
```
Comment (MarkRiddoch) [inline docs/conf.py]: OSIsoft or Dianomic?

File: docs/conf.py [hunk 3]
```diff
@@ -47,18 +45,18 @@
...
+author = u'Dianomic Systems'
```
Comment (MarkRiddoch) [inline docs/conf.py]: Was OSIsoft?

File: docs/conf.py [hunk 4]
```diff
@@ -31,8 +30,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.autodoc',
-    'sphinx.ext.ifconfig','sphinx.ext.todo']
+extensions = []
```
Comment (MarkRiddoch) [inline docs/conf.py]: Not sure if this is not a step backwards

File: benchmarks/fogbench/templates/fogbench_sensor_coap.template.json [hunk 5]
```diff
@@ -23,6 +23,8 @@
                         { "name": "z", "type": "number", "min": -255.0, "max": 255.0 } ] },
   { "name"          : "mouse",
     "sensor_values" : [ { "name": "button", "type": "enum", "list": [ "up", "down" ] } ] },
+  { "name"          : "mouse",
```
Comment (MarkRiddoch) [inline benchmarks/fogbench/templates/fogbench_sensor_coap.template.json]: Duplicate introduced

File: docs/Makefile [hunk 6]
```diff
@@ -3,22 +3,16 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = python -msphinx
```
Comment (MarkRiddoch) [inline docs/Makefile]: Was this an intentional change?

File: docs/Makefile [hunk 7]
```diff
@@ -3,22 +3,16 @@
...
 
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-livehtml:
-	sphinx-autobuild -b html "$(SPHINXOPTS)" "$(BUILDDIR)/html" --host "$(HOSTADDRESS)" --port "$(PORTNO)" --watch "$(CODEDIR)"
-
-.PHONY: help livehtml Makefile
+.PHONY: help Makefile
```
Comment (MarkRiddoch) [inline docs/Makefile]: Again I would question if this was an intentional change.

File: docs/conf.py [hunk 8]
```diff
@@ -76,61 +74,42 @@
...
 
 # -- Options for HTML output ----------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-import sphinx_rtd_theme
-html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme = 'sphinx_rtd_theme'
```
Comment (MarkRiddoch) [inline docs/conf.py]: Lost shine_rtd_theme

File: docs/conf.py (hunk 4 above)
Comment (izoratti-dianomic) [reply docs/conf.py]: I used the standard Read The Docs template. 
I think the misunderstanding here is that so far docs have been used to document the code, while here we have the documentation for the product.

Here is the important topic: If we want to maintain code documentation in /docs, then we must move the product documentation in a separate folder or even repository. We must make this decision asap.

File: docs/conf.py (hunk 2 above)
Comment (izoratti-dianomic) [reply docs/conf.py]: The documentation is owned by Dianomic
The code is owned by OSIsoft

File: docs/conf.py (hunk 3 above)
Comment (izoratti-dianomic) [reply docs/conf.py]: Code by OSIsoft
Documentation by Dianomic

File: docs/conf.py (hunk 8 above)
Comment (izoratti-dianomic) [reply docs/conf.py]: I used the default for Read the Docs - should I change it?

File: benchmarks/fogbench/templates/fogbench_sensor_coap.template.json (hunk 5 above)
Comment (izoratti-dianomic) [reply benchmarks/fogbench/templates/fogbench_sensor_coap.template.json]: It is done on purpose to make 10 rows.

File: docs/Makefile (hunk 6 above)
Comment (izoratti-dianomic) [reply docs/Makefile]: It is the default.

File: docs/Makefile (hunk 7 above)
Comment (izoratti-dianomic) [reply docs/Makefile]: Default from Read the docs.

**Pull request merged**